config/redis.yml

```diff
@@ -1,20 +1,20 @@
 default: &DEFAULT
-  db: 1
-  namespace: ararat
+  :db: 1
+  :namespace: measurements-api
```

Comment:
All right and proper, but what will it do to the running app?

Reply:
Should be nothing, since the namespace was overwritten with the correct value in every place it was used.